cdk/java/src/main/java/com/myorg/JavaStack.java

```diff
@@ -0,0 +1,53 @@
+package com.myorg;
+
+import software.constructs.Construct;
+
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.sam.CfnApplication;
+import software.amazon.awscdk.services.sam.CfnApplication.ApplicationLocationProperty;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JavaStack extends Stack {
```

Comment:
Should this class have a more specific name? Or is this a default we usually recommend for Java-based CDK projects?

See the C# pattern for example: https://github.com/alexcasalboni/aws-lambda-power-tuning/blob/master/cdk/csharp/src/Cdk/TheLambdaPowerTunerStack.cs

Reply:
It's the standard name, but I can rename so that it in line with the other CDK stacks.

Reply:
ok great 🚀